pkg/pool-weighted/contracts/WeightedMath.sol

```diff
@@ -243,14 +241,12 @@ library WeightedMath {
 
         // We can now compute how much extra balance is being deposited and used in virtual swaps, and charge swap fees
         // accordingly.
-        uint256 taxablePercentage = normalizedWeight.complement();
-        uint256 taxableAmount = amountInWithoutFee.mulUp(taxablePercentage);
+        uint256 taxableAmount = amountInWithoutFee.mulUp(normalizedWeight.complement());
         uint256 nonTaxableAmount = amountInWithoutFee.sub(taxableAmount);
 
         uint256 taxableAmountPlusFees = taxableAmount.divUp(FixedPoint.ONE.sub(swapFeePercentage));
```

Comment:
Not sure why we did not used `swapFeePercentage.complement()` there.

Reply:
Just didn't think of it. Saves 9 bytes